docs/getting-started/azure-1click.md

```diff
@@ -0,0 +1,270 @@
+# Mainnet/Testnet: Running Nodes using Azure 1-click Deployment
+
+This tutorial will use our Azure 1-click Deployment image to start and create the latest Croeseid Testnet validator or full node.
+
+## Step 1. Azure Account Creation
+
+You will first need to create an [Microsoft Azure](https://azure.microsoft.com/) account with a `Pay-As-You-Go` subscription. This will require providing your credit card information to `Microsoft Azure` and you may be subject to getting charged when you create a virtual machine.
+
+Please read `Microsoft Azure` free trial introduction to see if you are eligible for the free-tier.
+
+## Step 2. Common Setup
+
+### Step 2-1. Search for Crypto.org Chain on Marketplace
+
+Sign in to your Microsoft Azure account and go to [Marketplace](https://portal.azure.com/#blade/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home). Search for "Crypto.org Chain 1-Click Node".
+
+![](./assets/azure_marketplace.png)
+
+### Step 2-2. Create a Virtual Machine
+
+Choose the image and click "Create" to start creating the Crypto.org Chain 1-Click Node.
+
+#### 1. Basic
+
+![](./assets/azure_1click_basics.png)
+
+| Configuration           | Value                                                                                                            |
+| ----------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| _Size_                  | Select a recommended one or _DC2s_v3_                                                                            |
+| _SSH public key source_ | Choose _"existing public key"_                                                                                   |
+| _SSH public key_        | Copy and past your [SSH public key](https://docs.microsoft.com/azure/virtual-machines/linux/mac-create-ssh-keys) |
+| _Resource group_        | We suggest to create a new and dedicated one so that you can easily manage resources attached to the instance    |
```

Comment:
```suggestion
| _Resource group_        | We suggest creating a new and dedicated one so that you can easily manage resources attached to the instance    |
```

Reply:
fixed